src/forms/formHelpers.js

```diff
@@ -157,6 +157,9 @@ const FormBottomRow = function({ left, middle, right }) {
 */
 const FormPartsContainer = function({ title, clarifier, children, navData }) {
   return (
+    <Form
+    size='massive'
+    className='household-size-form flex-item flex-column'>
```

Comment:
Shoot, just noticed - we need to pass a custom class name in for the form - look at that first class name. Is that something you feel up to adjusting?

Reply:
We could possibly just use a changed version of the title to be the class name. Something like ``var formClass = title.toLowerCase().replace(` `, `-`);``